include/grpc/event_engine/event_engine.h

```diff
@@ -356,44 +361,35 @@ class EventEngine : public std::enable_shared_from_this<EventEngine> {
     using LookupTXTCallback =
         absl::AnyInvocable<void(absl::StatusOr<std::vector<std::string>>)>;
 
+    /// Implementations should make sure that all pending requests are cancelled
```

Comment:
Suggest just removing this comment; I think this information is already covered in the comment above.

Reply:
Done.